server/text_generation_server/habana_quantization_env.py

```diff
@@ -5,7 +5,9 @@
 
 assert "habana_frameworks" not in sys.modules
 
-is_quantization_enabled = os.getenv("QUANT_CONFIG", "") != ""
+use_inc = os.getenv("USE_INC", "0") == "1"
```

Comment:
use_inc=1 by default in OH. don't we want same in tgi?

Reply:
you are right, in final version we should have INC as a default